### DISCUSSION TITLE
今天和若藻一起过七夕~

### DISCUSSION THREAD

**Discussion by FireFlyLiuYing** [share121/inter-knot#949] [其他]
爱你，若藻~
![CID1552103_20240727132303](https://github.com/user-attachments/assets/2216f936-ac2d-4ef0-b023-21395b04a03f)

**Comment by aaa00073** [comment]
明明你的名字带了两个流萤！

**Comment by wugan-bot156** [comment]
好康(๑•̀ㅂ•́)و✧

**Comment by nightrain2333** [comment]
兄弟图真高清<img src="https://img-static.mihoyo.com/communityweb/upload/81ed7cd6675af50aec9cc622bda99b88.png" alt="比利_害羞" width="75" >

**Comment by bushigemengsishuihaiyinga** [comment]
我嘞个BA玩家无处不在啊
